PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/FunctionRunState.java

```diff
@@ -44,24 +72,144 @@ public FunctionRunState(@NotNull Project project, FunctionRunConfiguration funct
         this.functionRunConfiguration = functionRunConfiguration;
     }
 
+
     @Override
     protected String getDeployTarget() {
         return "null";
     }
 
+    private void launchDebugger(final Project project, int debugPort) {
+        final Runnable runnable = () -> {
+            final RunManagerImpl manager = new RunManagerImpl(project);
```

Comment:
RunManager runManager = RunManager.getInstance(project);

Reply:
`RunnerAndConfigurationSettingsImpl` need `RunManagerImpl` as parameter in constructor, `RunManager.getInstance(project)` returns `RunManager`